src/utils/code-parameters.ts

```diff
@@ -172,6 +188,22 @@ export const elementValueChangedCodeParameters = new CodeParametersBuilder({
           CodeEditorSuggestionItemKind.Method
         ),
         sectionsExpandedState: new CodeParameterItem<Record<string, boolean>>('Sections Expanded State'),
+        addSections: new CodeParameterItem<(sections: LayoutSection[]) => void>(
```

Comment:
I would change sections helpers to simplify working with them for end users.

1. Add Section and already assign elements for it
```
const elements = []; // create and save elements;

.addSection({ name: '1', elements: [element1.id] });
.addSection({ name: '2', elements: [element2.id] });
```

2. Get Section By Name - returns section object with elements
3. Remove From section
```
removeFromSection(section.id, [element1.id]);
```
4. Remove section
5. Add to section
```
addToSection(section.id, [element1.id])
```

In addition, I would create an object with all helpers for sections like `sectionsHelper: { add, get, getAll, update, assign, unassign, remove }`